geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -246,5 +252,27 @@ object RichSimpleFeatureType {
       Seq(GEOMESA_PREFIX) ++ userData[String](USER_DATA_PREFIX).map(_.split(",")).getOrElse(Array.empty)
 
     def userData[T](key: AnyRef): Option[T] = Option(sft.getUserData.get(key).asInstanceOf[T])
+
+    def getKeywords: java.util.Set[String] = {
+      userData[String](KEYWORDS_KEY).map(_.split(KEYWORDS_DELIMITER).toSet.asJava).getOrElse(java.util.Collections.emptySet[String])
```

Comment:
Alternatively, this line
```
 userData[String](KEYWORDS_KEY).toSet.flatMap(_.split(KEYWORDS_DELIMITER))
```
would do the same and allow for removing the ```import scala.collection.JavaConverters._```

I'd have a small preference for that since I prefer JavaConversions over JavaConverters;)